pkg/resource/logging-config/alloy/logging-config.alloy.yaml.template

```diff
@@ -33,6 +33,14 @@ alloy:
       # This is needed to allow alloy to create files when using readOnlyRootFilesystem
       - name: alloy-tmp
         mountPath: /tmp/alloy
+    # We decided to configure the alloy-logs resources as such after some investigation done https://github.com/giantswarm/giantswarm/issues/32655
+    resources:
+      limits:
+        cpu: 2
+        memory: 300Mi
```

Comment:
Does it make sense to set up a memory limit of 300Mi when the VPA is configured with a maxAllowed value of 1Gi ?

Reply:
VPA will scale limits as well